src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss

```diff
@@ -0,0 +1,228 @@
+.bar-wrapper {
+    width: 144px;
+    height: 12px;
+}
+
+.bar-state {
+    text-align: center !important;
+    .unknow-text {
+        margin-left: -5px;
+    }
+}
+
+.bar-state-chart {
+    margin-top: 2px;
+    .loop-height {
+        height: 2px;
+    }
+}
+
+.bar-state-error {
+    position: relative;
+    top: -4px;
+}
+
+.error-text {
+    position: relative;
+    top: 1px;
+    margin-left: -5px;
+    cursor: pointer;
+}
+
+.scanning-button {
+    height: 24px;
+    margin-top: 0;
+    margin-bottom: 0;
+    vertical-align: middle;
+    top: -12px;
+    position: relative;
+}
+
+.tip-wrapper {
+    display: inline-block;
+    height: 15px;
+    color: #fff;
+    text-align: center;
+    font-size: 10px;
+    line-height: 15px;
+}
+
+.tip-position {
+    margin-left: -4px;
+}
+
+.tip-block {
+    margin-left: -3px;
+}
+
+.bar-block-critical {
+    background-color: red;
+}
+
+.bar-block-high {
+    background-color: #e64524;
+}
+
+.bar-block-medium {
+    background-color: orange;
+}
+
+.bar-block-low {
+    background-color: #007CBB;
+}
+
+.bar-block-none {
+    background-color: green;
+}
+
+.bar-block-unknown {
+    background-color: grey;
+}
+
+.bar-tooltip-font {
+    font-size: 13px;
+    color: #ffffff;
+}
+
+.bar-tooltip-font-title {
+    font-weight: 600;
+}
+
+.bar-summary {
+    margin-top: 12px;
+    text-align: left;
+}
+
+.bar-scanning-time {
+    margin-top: 12px;
+}
+
+.bar-summary-item {
+    margin-top: 3px;
+    margin-bottom: 3px;
+}
+
+.bar-summary-item span:nth-child(1) {
+    width: 30px;
+    text-align: center;
+    display: inline-block;
+}
+
+.bar-summary-item span:nth-child(2) {
+    width: 28px;
+    display: inline-block;
+}
+
+.option-right {
+    padding-right: 16px;
+}
+
+.refresh-btn {
+    cursor: pointer;
+}
+
+.refresh-btn:hover {
+    color: #007CBB;
+}
+
+.label.label-medium {
+    background-color: #ffe4a9;
+    border: 1px solid orange;
+    color: orange;
+}
+
+.tip-icon-medium {
+    color: orange;
+}
+
+.label.label-low {
+    background: rgba(251, 255, 0, 0.38);
+    color: #c5c50b;
+    border: 1px solid #e6e63f;
+}
+
+.tip-icon-low {
+    color: #007CBB;
+}
+
+.font-color-green {
+    color: green;
+}
+
+.bar-tooltip-font-larger span {
+    font-size: 16px;
+    vertical-align: middle
+}
+
+hr {
+    border-bottom: 0;
+    border-color: #aaa;
+    margin: 6px -10px;
+}
+
+.font-weight-600 {
+    font-weight: 600;
+}
+
+.rightPos {
```

Comment:
this rightPos style is not needed. in _mixin.scss there is a grid-right-top-pos mixin you can directly introduced it in your component.

Reply:
fixed